mgr/src/main.rs

```diff
@@ -272,6 +256,62 @@ fn system_info(args: &SystemInfoArgs, config: &Config) -> Result<()> {
     Ok(())
 }
 
+fn restart(args: &GeneralArgs, config: &Config) -> Result<()> {
```

Comment:
I think this is just giving users a gun to shoot themselves in the foot with.

Reply:
Not really.  
Do we have any guarantee that the service will never have an issue and the service will never need to restart? 
Does every time a restart-solvable issue need us as 7x24 support to do it and disallow the user to do it?

By the way, not all user has knowledge of Linux operating to restart the services, so some user needs this.  You also have trouble on restarting the service, and you can use `kld-mgr restart` to restart the service on your node.